test/inDelta.js

```diff
@@ -0,0 +1,26 @@
+var tape = require("tape");
```

Comment:
Surprising to see `var`.

Reply:
that file was just copied from other d3 repos :)